xml/System.Net.Http/HttpRequestOptions.xml

```diff
@@ -845,10 +845,10 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="value" Type="System.Object" RefType="out" Index="1" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The key to locate.</param>
+        <param name="value">When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the <paramref name="value" /> parameter. This parameter is passed uninitialized.</param>
+        <summary>Gets the value that is associated with the specified key.</summary>
+        <see langword="true" /> if the <xref:System.Net.Http.HttpRequestOptions> instance contains an element with the specified key; otherwise, <see langword="false" />.</returns>

```

Comment:
xref is reserved for usage in markdown. In plain xml, we use see cref:
```suggestion
        <see langword="true" /> if the <see cref="T:System.Net.Http.HttpRequestOptions"/> instance contains an element with the specified key; otherwise, <see langword="false" />.</returns>

```